include/fmt/core.h

```diff
@@ -740,19 +740,26 @@ inline auto get_container(std::back_insert_iterator<Container> it)
 }
 
 template <typename Char, typename InputIt, typename OutputIt>
-FMT_CONSTEXPR auto copy_str(InputIt begin, InputIt end, OutputIt out)
+FMT_CONSTEXPR auto copy_str_constexpr(InputIt begin, InputIt end, OutputIt out)
     -> OutputIt {
   while (begin != end) *out++ = static_cast<Char>(*begin++);
   return out;
 }
```

Comment:
I suggest merging this into the first overload of `copy_str` as before. The second overload can continue calling it as `copy_str<Char, const Char*, Char*>(...)`.

Reply:
I thought it would lead to recursion, but it doesn't